src/components/ToggleSwitch/index.tsx

```diff
@@ -63,12 +63,12 @@ const styles = StyleSheet.create({
   mainStyes: {
     position: 'relative',
     borderRadius: dimen.normalizeDimen(12),
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.almostBlack,
     height: dimen.normalizeDimen(20),
     width: dimen.normalizeDimen(42),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color of the main style has been changed from `COLORS.lightgrey` to `COLORS.almostBlack`. Ensure that this color change aligns with the overall theme and accessibility standards of your application. Dark colors can sometimes be hard to see for users with visual impairments.
```diff
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.almostBlack,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->